hrm-domain/lambdas/custom-integrations/uscr-dispatcher/logger.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+export const logger = ({
```

Comment:
I don't think there's any point in this, we can already filter logs from a specific lambda or lambda type in Datadog and they are filterable by severity (it's only the HRM _service_ logs where this isn't the case). Unless there is some other contextual information you'd like to add to every log statement, like contact ID or something, it's better to just use the native loggers directly